Formula/Pelikan.rb

```diff
@@ -0,0 +1,22 @@
+class Pelikan < Formula
+  desc "Pelikan: production-ready cache services"
```

Comment:
Please remove the `Pelikan: ` from the description. When appearing in a context where multiple formulae are listed, Homebrew will automatically prefix the formula name. In other contexts (like `brew info`) this prefix would look weird.